boards/arm/volteer/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_VOLTEER
```

Comment:
You are your thoughts about working on the downstream version of volteer for now until we get things working. Then we will definitely want to upstream this board.

Reply:
ACK, I will delete this CL